### ISSUE TITLE
Remove search plugin from topbar

### ISSUE THREAD

**Issue by vvmruder** [qgis/qwc2-demo-app#320]
I tried to remove the search from the TopBar plugin via appConfig.js This dont work since there is no conditional rendering on this search element. Its mandatory.

What would be the suggested way to achieve this. Creating a complete custom TopBar plugin?

**Comment by manisandro** [comment]
Yes - if the topbar styling does not need any adjustments, then looks good.

**Comment by vvmruder** [comment]
Only thing is, the items right next to it are sliding to the most left than. This looks a little strange. Any suggestion how to handle that in a easy way?

**Comment by vvmruder** [comment]
Lets discuss on the PR.
